HouseRules_Essentials/Rules/AbilityDamageOverriddenRule.cs

```diff
@@ -9,7 +9,7 @@
 
     public sealed class AbilityDamageOverriddenRule : Rule, IConfigWritable<Dictionary<AbilityKey, List<int>>>, IMultiplayerSafe
     {
-        public override string Description => "Ability damage values are overridden";
+        public override string Description => "Ability damage is adjusted";
```

Comment:
In the past Mex has preferred the rule descriptions to match the rule names - An 'adjust' rule is adding a +_ modifier to an existing value, whereas an override rule sets a value directly. I don't have strong feelings on this myself, but thought I'd mention as I've been asked to change from 'adjusted' to 'overridden' in the descriptions before.